scripts/prepare_automatic_backups.py

```diff
@@ -44,6 +54,15 @@ def generate_backup_url(cloud_storage_bucket_name, module_class_names):
 
 
 def update_cron_dict(cron_dict):
+    """Adding new urls in dict for backup.
+
+    Args:
+        cron_dict: dict(str ,str). Content of yaml file in dictionary type.
+            'description' : 'weekly bakcup'
```

Comment:
Follow this format instead, so that the typeinfo for each value is clear. (Replace the {{...}} parts with the correct descriptions.):

```
... Content of yaml file in dictionary type. The keys and values are as follows:
    'description': str. {{Description of what this field means.}}
    'url': str. {{Description of what the URL represents.}}
    ...
```

Reply:
done it.